lib/origen_testers/generator.rb

```diff
@@ -254,8 +254,16 @@ def reference_file
     end
 
     def import(file, options = {})
+      options = {
+        component: true # by default import method treats file as a component flow file e.g. _my_component.rb
+      }.merge(options)
```

Comment:
I think this goes against what @ginty had in mind-- that it should just work whether the user is importing a component or a sub-flow.  Here it is required that user know to set `component: false` when using `import` for a sub-flow. 

I think to provide the "it just works" functionality you would have to check to see whether file exists or not using the filename as-is but WITH the underscore.  If not, then check it also without the underscore.  This way if there are 2 files, one with underscore and one without, the underscore one would take priority.  

With that, then no passing of option is needed.

Reply:
yes, agreed

Reply:
okay i misunderstood i think, thanks

Reply:
@ginty @redxeth , let me know if this makes sense, i've upshed in some new changes.